cloudbuild.yaml

```
@@ -14,12 +14,36 @@
 #
 
 steps:
-###########################################################
-# Step 1: Deploy
-###########################################################
+  ############################################################
+  # Step 0: Initialize git repo to enable checking for changes
+  ############################################################
+- name: 'gcr.io/cloud-builders/git'
+  entrypoint: 'bash'
+  args:
+    - '-c'
+    - |
+      git init
+      git config user.email "builder@bigquery-utils.repo"
+      git config user.name "builder"
+
+      git commit -m "empty commit"
+      git remote add origin https://github.com/GoogleCloudPlatform/bigquery-utils.git
+      git fetch origin master
+
+      # Fetch all PRs to get history for PRs created from forked repos
+      git fetch origin +refs/pull/*/merge:refs/remotes/origin/pr/*
```

TIL  you can do this for just a single PR (rather than all PRs)
source: https://gist.github.com/piscisaureus/3342247
```suggestion
      git fetch origin +refs/pull/${_PR_NUMBER}/merge:refs/remotes/origin/pr/${_PR_NUMBER}
```